whisper_server/whisper_server.py

```
@@ -17,79 +17,75 @@
-            "response_format": "json"
-        }
-        
+        files = {"file": audio_file}
+        data = {"model": model, "response_format": "json"}
+
         logger.info(f"Transcribing {file_path} with model {model}")
         start_time = time.time()
-        
+
         try:
             response = requests.post(url, headers=headers, files=files, data=data)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)